src/mono/wasm/Makefile

```diff
@@ -176,6 +176,9 @@ run-tests-%:
 run-browser-tests-%:
 	PATH="$(CHROMEDRIVER):$(PATH)" XHARNESS_COMMAND=test-browser $(DOTNET) build $(TOP)/src/libraries/$*/tests/ /t:Test /p:TargetOS=Browser /p:TargetArchitecture=wasm /p:Configuration=$(CONFIG) $(MSBUILD_ARGS)
 
+run-browser-outerloop-tests-%:
+	PATH="$(CHROMEDRIVER):$(PATH)" XHARNESS_COMMAND=test-browser $(DOTNET) build $(TOP)/src/libraries/$*/tests/ /t:Test /p:TargetOS=Browser /p:TargetArchitecture=wasm /p:Configuration=$(CONFIG) /p:OuterLoop=true $(MSBUILD_ARGS)
```

Comment:
We should add a
`MSBUILD_WASM_BUILD_ARGS="/p:TargetOS=Browser /p:TargetArchitecture=wasm /p:Configuration=$(CONFIG)"`

.. and replace all the instances of that string on the command lines, with this variable. There will be 5, including this new one.

Reply:
We need to write up some documentation on all these things, the Makefile shouldn't be a substitute for docs

Reply:
Docs on how to build, and run various kinda tests?

Reply:
Something as simple as a readme.md in the wasm directory that describes the xharness options and mentions the properties that might be useful

Reply:
I have opened a PR #45863 for the README.